### DISCUSSION TITLE
kuwo歌源等你下课 (with 杨瑞代)不匹配

### DISCUSSION THREAD

**Discussion by zzz6839** [1715173329/UnblockNeteaseMusic#475] [Q&A]
kuwo因为有着很多优质的音源所以无法被替换，其他歌曲与原版有出入无所谓，但是这首歌是周杰伦的，就很难受，想问下有没有其他解决方法。

log:
[32mINFO[39m: [36m[1m[32m(provider/match)[0m[36m Replaced: [531051217] 等你下课 (with 杨瑞代)[39m
    audioId: 531051217
    songName: "等你下课 (with 杨瑞代)"
    url: "http://sr.sycdn.kuwo.cn/e2b860bf27a3f650dd2324d2a9f6c8a4/61e6f1ec/resource/n1/51/38/3611079855.mp3"

**Comment by 1715173329** [comment]
#376

**Reply by pan93412** [reply]
其實應該是 #428
